src/gui/qgsrelationeditorwidget.cpp

```diff
@@ -30,11 +30,74 @@
 #include "qgslogger.h"
 #include "qgsvectorlayerutils.h"
 #include "qgsmapcanvas.h"
+#include "qgsvectorlayerselectionmanager.h"
 
 #include <QHBoxLayout>
 #include <QLabel>
 #include <QMessageBox>
 
+/// @cond PRIVATE
+
+class QgsFilteredSelectionManager : public QgsVectorLayerSelectionManager
```

Comment:
Can you add some docstrings explaining the rational for this class

Reply:
Done